odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1203,9 +1203,14 @@ def _prepare_qcontext(self):
         """ Returns the qcontext : rendering context with website specific value (required
             to render website layout template)
         """
+        user_id = self.env["res.users"].browse(self.env.user.id)
+        cids = [str(user_id.company_id.id)]
```

Comment:
`cids` gives no clue of what it holds :-)
If this were to stay, rename it `company_ids`

Reply:
`cids` back to life :D
We usually use incomprehensible and very short cookies name to lighter the data send from the client, as the cookies are sent on every http request and there is no cache mechanism, so any gain is good.
But for a variable name, it really should be avoided :) `cids` doesn't mean anything

Reply:
ahah my apologies, I forget to answer you. At first, I wanted to match this https://github.com/odoo/odoo/blob/4c1271738daa039e79a11646fef3f4264e4d96d2/addons/mail/controllers/main.py#L77 
Thanks for the clarification!